### ISSUE TITLE
Nightly fails npm install related to link-hooks.js

### ISSUE THREAD

**Issue by jbrantly** [Microsoft/TypeScript#4966]
I'm guessing this is related to #4961 

```
npm install typescript@next
> typescript@1.7.0-dev.20150925 postinstall /home/jbrantly/ts-loader/node_modules/typescript
> node scripts/link-hooks.js


module.js:340
    throw err;
          ^
Error: Cannot find module '/home/jbrantly/ts-loader/node_modules/typescript/scripts/link-hooks.js'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:929:3
npm ERR! Linux 3.13.0-44-generic
npm ERR! argv "node" "/usr/bin/npm" "install" "typescript@next"
npm ERR! node v0.10.36
npm ERR! npm  v2.14.1
npm ERR! code ELIFECYCLE

npm ERR! typescript@1.7.0-dev.20150925 postinstall: `node scripts/link-hooks.js`
npm ERR! Exit status 8
npm ERR! 
npm ERR! Failed at the typescript@1.7.0-dev.20150925 postinstall script 'node scripts/link-hooks.js'.
npm ERR! This is most likely a problem with the typescript package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node scripts/link-hooks.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls typescript
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/jbrantly/ts-loader/npm-debug.log
```

This happened both on my machine and in a travis-ci build.

**Comment by vladima** [comment]
currently npm ignores `script` folder (via .npmignore)

**Comment by aaronryden** [comment]
I am also seeing this on latest

**Comment by mhegazy** [comment]
@weswigham can u take a look. I would rather we remove the post install script. The file is only absent on a new git clone.

**Comment by weswigham** [comment]
Mmm... I should just make the post install silent if the script is not present. Did not realize scripts wasn't part of the npm package. I'll get right on that. @mhegazy: its absent after a new git clone, but its potentially mismatched after every checkout, hence why its regenerated with a post checkout script, keeping the editor experience up to date.

**Comment by weswigham** [comment]
Should be fixed by #4973.
